src/sentry/web/frontend/debug/mail.py

```diff
@@ -210,6 +212,21 @@ def make_performance_event(project):
     return perf_event
 
 
+def make_generic_event(project):
+    occurrence, group_info = process_event_and_issue_occurrence(
+        TEST_ISSUE_OCCURRENCE.to_dict(),
+        {
+            "event_id": uuid.uuid4().hex,
+            "fingerprint": ["group-1"],
```

Comment:
No need to include a fingerprint here, the fingerprint we care about is on the occurrence.